modules/standard/JSON.chpl

```diff
@@ -96,85 +179,185 @@ module JSON {
         writer.write(field);
       }
 
+      /*
+        Start serializing a nested class inside the current class. In this format
+        inheritance is not represented and parent fields are printed before child
+        fields. For example, the following classes with values ``x=5`` and
+        ``y=2``:
+
+        .. code-block:: chapel
+
+          class Parent {
+            var x : int;
+          }
+
+          class Child: Parent {
+            var y : int;
+          }
+
+        would be serialized as:
+
+        .. code-block:: text
+
+          {"x":5, "y":2}
+
+        :arg writer: The ``fileWriter`` to be used when serializing. Must match
+          writer used to create current AggregateSerializer.
+        :arg name: The name of the class type.
+        :arg size: The number of fields in the class.
+
+        :returns: A new AggregateSerializer
```

Comment:
A new :type:`AggregateSerializer`

Reply:
There's actually a chpldoc bug with this one, unfortunately, so the ``:type`` won't work.